content/en/docs/reference/glossary/cloud-provider.md

```diff
@@ -1,18 +1,32 @@
 ---
-title: Cloud Provider
+title: Cloud provider
 id: cloud-provider
 date: 2018-04-12
 full_link: /docs/concepts/cluster-administration/cloud-providers
 short_description: >
-  Cloud provider is a company that offers cloud computing platform that can run Kubernetes clusters.
+  A company that offers a cloud computing platform
```

Comment:
I could be wrong but I have a different understanding of "cloud provider".
When we talking about cloud providers in the context of kubernetes, we are talking about azure, aws, openstack, gce etc. Some of them do have a single company behind them while others don't. A cloud provider is somehow an ambiguous word to me (a non-native speaker) because it is sometimes used to refer to a *technology* when we talk about the type of cloud which can be leveraged by Kubernetes, and it is also used to refer to a specific *vendor* that sells such type of a cloud service.

Reply:
I'm a native English speaker; I live in England. _Cloud provider_ doesn't to me imply “provider” in the sense of a technology. An internet provider or internet service provider is a firm who hooks you up to the internet and bills you for it.

Years ago, what we today call “the cloud” was called _utility computing_ but now the _cloud (computing)_ name is widely known and used. So the following examples are all cloud providers / cloud service providers:
- Amazon Web Services
- DigitalOcean
- Google (Google Cloud Platform)
- IBM (IBM Cloud)
- Kingsoft 
- Microsoft (Azure)
- Packet 
- Platform9
- Rackspace
- Tencent (Tencent Cloud)

and, because they like to stay in business, they are all vendors as well.
I think the original and revised sense of this glossary entry is correct. If there's a risk of confusion, I hope we can make things clearer through this entry itself.

Reply:
Right. There are many companies providing clouds suitable for k8s. But, from a k8s user/admin's perspective, I'd focus more on the interface between k8s and the provider. Even when we are talking about a cloud which is very specific to a company, we talk about the tech rather than the company. See https://github.com/kubernetes/cloud-provider/blob/master/plugins.go, we discuss 'aws' rather than Amazon, 'gce' instead of Google, it is 'vsphere' the provider that is deprecated, not VMware. Hope this slight difference can be somehow considered.

Reply:
So there is also:
- SIG Cloud Provider
- [kubernetes/cloud-provider](https://github.com/kubernetes/cloud-provider/) (“the cloud-provider interface”)

and perhaps the full glossary entry should mention these two?